react/features/overlay/components/OverlayContainer.js

```diff
@@ -1,12 +1,38 @@
-/* global APP */
-
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import PageReloadFilmStripOnlyOverlay from './PageReloadFilmStripOnlyOverlay';
 import PageReloadOverlay from './PageReloadOverlay';
+import SuspendedFilmStripOnlyOverlay from './SuspendedFilmStripOnlyOverlay';
 import SuspendedOverlay from './SuspendedOverlay';
+import UserMediaPermissionsFilmStripOnlyOverlay
+    from './UserMediaPermissionsFilmStripOnlyOverlay';
 import UserMediaPermissionsOverlay from './UserMediaPermissionsOverlay';
 
+declare var APP: Object;
+declare var interfaceConfig: Object;
+
+/**
+ * Maps the overlay component that have to be displayed depending on the
+ * filmstrip only mode.
+ *
+ * @type {object}
+ */
+const filmStripOnlyModeToComponent = {
+    pageReload: {
+        filmStripOnly: PageReloadFilmStripOnlyOverlay,
```

Comment:
So how it is going to look like. Do you mean:
```
const pageReload = Symbol('pageReload');
const filmStripOnlyModeToComponent = {
    pageReload: {....

```

and then accessing it as `filmStripOnlyModeToComponent[pageReload]` or something else?